content/en/docs/storage-providers/seal-workers.md

```diff
@@ -221,3 +221,56 @@ The only case where running multiple workers per machine may be a good idea is w
 When using proprietary Nvidia drivers, it's possible to select which GPU device will be used by Lotus with the `NVIDIA_VISIBLE_DEVICES=[device number]` env var.
 
 Device numbers can be obtained with the `nvidia-smi -L` command.
+
+#### Sector Storage Groups
+
+The sectorstore.json contains two additional optional fields to allow for creating worker groups and avoiding unnecessarily moving data between multi-purpose workers.
+
+##### Use case
+
+- This feature is useful when storage path is not shared between the worker.
+- This feature can used to group workers if some of the workers share a storage path (ex: NFS) but not all.
+- In case all workers share the same storage path like NFS or similar storage then this feature would not be helpful.
+- This feature does not relate to long term storage path.
```

Comment:
```suggestion
- This feature is useful when the Lotus storage path is not shared between workers.
- This feature can used to group workers together if some, but not all, of them share a storage path (e.g. NFS). If all the workers share the same storage path, then this feature should not be used.
- This feature does not relate to long term storage path.
```

Reply:
In the second point - This feature can be used